pytest-wdl-tests/tests/expr/versioning/version_one/test_versioning_version_one.py

```diff
@@ -0,0 +1,5 @@
+def test_versioning(workflow_runner):
+    """
+    First line of the WDL file must be a version statement
```

Comment:
Actually, I think we say the first *active* line of a WDL file should be a version statement.

We might want to test that we can have newlines and comments before the version declaration